mysql-test/suite/rpl/t/rpl_percona_bug995624.test

```diff
@@ -0,0 +1,46 @@
+source include/master-slave.inc;
+source include/not_gtid_enabled.inc;
+
+connection slave;
+set global userstat=ON;
+--echo Rows_updated on slave - starting situation
+select ROWS_UPDATED from information_schema.client_statistics;
+
+connection master;
+set global userstat=ON;
```

Comment:
The bug is not about the master, so strictly speaking this is redundant. But, it could be a good idea to check that the userstat rows_updated on master and slave increase by the same amount